Dockerfile

```
@@ -150,7 +104,7 @@ ARG APP_NAME
 # Copy over built executable into a fresh container
 COPY --from=builder /workdir/build/bin/${APP_NAME} /usr/bin/${APP_NAME}
 
-# TODO: We should un hood this part, its very specific 
+# TODO: We should un hood this part, its very specific
 # to our kurtosis setup.
 RUN mkdir -p /root/jwt /root/kzg && \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)